SubmarineDownloader/Lua/Autorun/submarinedownloader.lua

```diff
@@ -1,6 +1,9 @@
+-- Submarine Downloader By
+-- Evil Factory & Greetings (AtFreezingPoint)
+-- Workshop Collection is Outsourced; credit to submarine authors.
 -- put here the id of all workshop collections you want the script to download from.
--- example: {"2808536198"}
-local workshopCollections = {}
+-- example: {"2963343754"}
+local workshopCollections = {2963343754}
```

Comment:
Can you leave the table empty, so anyone using it add their own collection instead of getting confused that it's downloading from some collection that they don't know